No functional changes. Only Docstring changes and formatting. removed…

malcolm-dsider opened the pull request:
… doc-author and "param self"; added type and rtype.

softwareengineerprogrammer commented on tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt:
```diff
@@ -1,72 +1,203 @@
-
-*** Simulation Parameters ***
-Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
+                               *****************
```
Seems like this was an accident? (input file content replaced with output content and output file deleted)

softwareengineerprogrammer reviewed (changes requested):
Looks good - one issue with test files as commented

softwareengineerprogrammer commented:
@malcolm-dsider To address the unintended changes to the Wanju files, you can do the following:

```
git co e3c6c4c79425976085a7d939e4e00a308146a1b4 -- tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.out tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt

git commit -m "Revert unintended changes to Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery test files"
```

(And then `git push`)

malcolm-dsider commented:
Jonathan – 

 

Thanks for your help on this – It ws an error to commit that  output file as an input file, and I wasn’t certain how to undo the error, so I appreciate the advice….

 

But the first command seems to fail. There is no “git co” command???

 

Thanks

 

-Malcolm

 

From: Jonathan Pezzino ***@***.***> 
Sent: Friday, November 10, 2023 12:49 PM
To: NREL/python-geophires-x ***@***.***>
Cc: Malcolm Ross ***@***.***>; Mention ***@***.***>
Subject: Re: [NREL/python-geophires-x] No functional changes. Only Docstring changes and formatting. removed… (PR #45)

 

@malcolm-dsider <https://github.com/malcolm-dsider>  To address the unintended changes to the Wanju files, you can do the following:

git co e3c6c4c79425976085a7d939e4e00a308146a1b4 -- tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.out tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt
 
git commit -m "Revert unintended changes to Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery test files"

(And then git push)

—
Reply to this email directly, view it on GitHub <https://github.com/NREL/python-geophires-x/pull/45#issuecomment-1806262441> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AWGVYTYI2FZP342H2NCB4L3YDZZK5AVCNFSM6AAAAAA7GRWMJWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMBWGI3DENBUGE> .
You are receiving this because you were mentioned.  <https://github.com/notifications/beacon/AWGVYT275UYWUCEQGGRDLWDYDZZK5A5CNFSM6AAAAAA7GRWMJWWGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTTLVFQKS.gif> Message ID: ***@***.*** ***@***.***> >

malcolm-dsider commented:
Never mind – figured it out co=checkout

 

From: ***@***.*** ***@***.***> 
Sent: Friday, November 10, 2023 2:00 PM
To: 'NREL/python-geophires-x' ***@***.***>; 'NREL/python-geophires-x' ***@***.***>
Cc: 'Mention' ***@***.***>
Subject: RE: [NREL/python-geophires-x] No functional changes. Only Docstring changes and formatting. removed… (PR #45)

 

Jonathan – 

 

Thanks for your help on this – It ws an error to commit that  output file as an input file, and I wasn’t certain how to undo the error, so I appreciate the advice….

 

But the first command seems to fail. There is no “git co” command???

 

Thanks

 

-Malcolm

 

From: Jonathan Pezzino ***@***.***> 
Sent: Friday, November 10, 2023 12:49 PM
To: NREL/python-geophires-x ***@***.***>
Cc: Malcolm Ross ***@***.***>; Mention ***@***.***>
Subject: Re: [NREL/python-geophires-x] No functional changes. Only Docstring changes and formatting. removed… (PR #45)

 

@malcolm-dsider <https://github.com/malcolm-dsider>  To address the unintended changes to the Wanju files, you can do the following:

git co e3c6c4c79425976085a7d939e4e00a308146a1b4 -- tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.out tests/examples/Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery.txt
 
git commit -m "Revert unintended changes to Wanju_Yuan_Closed-Loop_Geothermal_Energy_Recovery test files"

(And then git push)

—
Reply to this email directly, view it on GitHub <https://github.com/NREL/python-geophires-x/pull/45#issuecomment-1806262441> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AWGVYTYI2FZP342H2NCB4L3YDZZK5AVCNFSM6AAAAAA7GRWMJWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQMBWGI3DENBUGE> .
You are receiving this because you were mentioned.Message ID: ***@***.*** ***@***.***> >

softwareengineerprogrammer commented:
> There is no “git co” command???

Yes, my apologies. (I noticed that after I commented and edited hoping to avoid the confusion but seems I wasn't quick enough.)

`co` is a very common alias for `checkout`, and I forgot not everyone has it. You can configure it by running:

```
git config --global alias.co checkout
```

which I would recommend.

Glad you got it figured out!

softwareengineerprogrammer reviewed (commented):
2nd revision looks good (retroactively approved post-merge for posterity's sake)
